lib/Doctrine/ruleset.xml

```diff
@@ -205,18 +205,18 @@
     <rule ref="SlevomatCodingStandard.Commenting.RequireOneLinePropertyDocComment"/>
     <!-- Forbid assignments in conditions -->
     <rule ref="SlevomatCodingStandard.ControlStructures.AssignmentInCondition"/>
-    <!-- Require consistent spacing for control structures -->
-    <rule ref="SlevomatCodingStandard.ControlStructures.ControlStructureSpacing">
```

Comment:
Control structures should still be spaced

Reply:
Definitely.

I guessed it was not enabled before as I think it was actually being overridden and not extended for `return` here https://github.com/doctrine/coding-standard/commit/fe9849cbb650e604b663c9eae940500d7f853565#diff-1d794c8c2cf3bf4d562ddaea8be99bb0R208-R210 but I may be wrong.

Also, no related tests were broken by removing it.

I will add in in another PR.

Reply:
Implemented here https://github.com/doctrine/coding-standard/pull/137